www/core/sts/list_sts.xml

```diff
@@ -13,14 +13,15 @@
 	<extension name="Joomla" element="joomla" type="file" version="3.10.12" targetplatformversion="3.7" detailsurl="https://update.joomla.org/core/extension.xml" />
 	<extension name="Joomla" element="joomla" type="file" version="3.10.12" targetplatformversion="3.8" detailsurl="https://update.joomla.org/core/extension.xml" />
 	<extension name="Joomla" element="joomla" type="file" version="3.10.12" targetplatformversion="3.9" detailsurl="https://update.joomla.org/core/extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="4.3.4" targetplatformversion="3.10.12" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="4.3.4" targetplatformversion="3.10.13" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="3.10.12" targetplatformversion="3.10" detailsurl="https://update.joomla.org/core/extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="4.3.4" targetplatformversion="4.0.4" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="4.3.4" targetplatformversion="4.0.5" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="4.3.4" targetplatformversion="4.0.6" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="4.4.0" targetplatformversion="3.10.12" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="4.4.0" targetplatformversion="3.10.13" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="4.4.0" targetplatformversion="4.0.4" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />

```

Comment:
```suggestion
	<extension name="Joomla" element="joomla" type="file" version="3.10.12" targetplatformversion="3.10" detailsurl="https://update.joomla.org/core/extension.xml" />
	<extension name="Joomla" element="joomla" type="file" version="4.4.0" targetplatformversion="4.0.4" detailsurl="https://update.joomla.org/core/sts/extension_sts.xml" />
```

We need this line back to allow updates within the 3.10 series into 3.10.12

Reply:
The diff looks confusing but is correct the existing line needs to stay and this needs to be added back:
<extension name="Joomla" element="joomla" type="file" version="3.10.12" targetplatformversion="3.10" detailsurl="https://update.joomla.org/core/extension.xml" />

Reply:
Correct.